utils/cargo-gbuild/tests/smoke.rs

```diff
@@ -0,0 +1,69 @@
+// This file is part of Gear.
```

Comment:
for testing you can just use gtest or if you want to make it as minimal as it could be than just Code::try_new

Reply:
and it will no longer require node!

Reply:
f223be2